pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -388,7 +388,7 @@ func (exp *flowExporter) sendTemplateSet(isIPv6 bool) (int, error) {
 	if err := exp.ipfixSet.PrepareSet(ipfixentities.Template, templateID); err != nil {
 		return 0, err
 	}
-	err := exp.ipfixSet.AddRecord(elements, templateID)
+	err := exp.ipfixSet.AddRecord(elements, 0, templateID)
```

Comment:
it would have been good (or it would be good if it's not too late) to define 2 different methods in go-ipfix: `AddRecord` and `AddRecordWithExtraElements`. It would avoid these cryptic changes in Antrea and it's probably good to keep backward compatibility for your library when you can...

Reply:
Good suggestion.